android/src/main/java/cl/json/RNSharePathUtil.java

```diff
@@ -57,7 +57,7 @@ public static Uri compatUriFromFile(@NonNull final ReactContext context, @NonNul
                     break;
                 }
             } catch (Exception e) {
-
+                System.out.println("ERROR " + e.getMessage());
```

Comment:
This would be better with some prefix indicating where it came from, e.g. 
```suggestion
                System.out.println("RNSharePathUtil::compatUriFromFile ERROR " + e.getMessage());
```

Reply:
> This would be better with some prefix indicating where it came from, e.g. 
> ```suggestion
>                 System.out.println("RNSharePathUtil::compatUriFromFile ERROR " + e.getMessage());
> ```

Agree with that.